src/ui/common/src/components/pages/operator/id/index.tsx

```diff
@@ -176,18 +184,8 @@ const OperatorDetailsPage: React.FC<OperatorDetailsPageProps> = ({
       }
     }
 
-    // TODO: only call this when we are inside a function operator. not an extract or load operator.
```

Comment:
slightly clean up this code by directly render the spec using code viewer. Thus, there's no need to set file for extract here